### ISSUE TITLE
Doesn't work with `sorbet-rails`

### ISSUE THREAD

**Issue by ghiculescu** [Shopify/tapioca#33]
You guys must be sick of me :) 

Have a look at https://gist.github.com/ghiculescu/eed3ba99051ce053d18268e4beb75d07. The first file is what's in our codebase from running `srb rbi gems` ages ago. The second file is what `tapioca` is generating for `sorbet-rails`.

When running `srb tc` we get lots of errors like this:

```
sorbet/rails-rbi/pluck_to_tstruct.rbi:8: <todo sym> cannot be used for type member ::ITypeAssert::Elem https://srb.help/7028
     8 |      ta_struct: ITypeAssert[T.type_parameter(:U)],
                                     ^^^^^^^^^^^^^^^^^^^^
    sorbet/rails-rbi/pluck_to_tstruct.rbi:8: <todo sym> is not a subtype of <todo sym>
     8 |      ta_struct: ITypeAssert[T.type_parameter(:U)],
                                     ^^^^^^^^^^^^^^^^^^^^

sorbet/rails-rbi/parameters.rbi:8: <todo sym> cannot be used for type member ::ITypeAssert::Elem https://srb.help/7028
     8 |    params(key: Symbol, ta: ITypeAssert[T.type_parameter(:U)]).
                                                ^^^^^^^^^^^^^^^^^^^^
    sorbet/rails-rbi/parameters.rbi:8: <todo sym> is not a subtype of <todo sym>
     8 |    params(key: Symbol, ta: ITypeAssert[T.type_parameter(:U)]).
                                                ^^^^^^^^^^^^^^^^^^^^
```

I think https://gist.github.com/ghiculescu/eed3ba99051ce053d18268e4beb75d07#file-tapioca-L27 is the culprit. There isn't an equivalent line in the `srb rbi gems` version.

-----------

Note that sorbet-rails bundles its own rbis that also cover this https://github.com/chanzuckerberg/sorbet-rails/tree/master/lib/bundled_rbi - so skipping this in some way might be an option. It's not a good idea to skip the entire `sorbet-rails` gem though because then none  of the generator code works.

**Comment by manhhung741** [comment]
> I think https://gist.github.com/ghiculescu/eed3ba99051ce053d18268e4beb75d07#file-tapioca-L27 is the culprit. There isn't an equivalent line in the srb rbi gems version.

Have you tried removing that line? Does the error go away?

**Comment by ghiculescu** [comment]
Can't remember, but I suspect yes. The problem is that every time you regenerate the rbi (eg. because you updated the gem) that line comes back.

**Comment by paracycle** [comment]
Apologies for the delay in getting to this. I'll take a look at this today. I guess `tapioca` should not be generating any constants that are of type `T::Types::TypeMember` anyway. I can filter those out.

**Comment by paracycle** [comment]
This was released in version 0.3.0
